README.markdown

```diff
@@ -7,19 +9,19 @@ Sublime Notebook is an attempt to use Sublime Text as a complete note taking app
 > Blog post for v0.3.0 (no longer recommended) https://medium.com/aviaryan/b8d846c47905#.hy8alq2ip
 
 
-## Why did you build this?
+## Why did you build this? :thinking:
 
 I have been note-taking for as long as I started using computers. I use notes mostly for technical stuff, but these days I am using notes to record all kinds of information like journals, ideas, snippets etc. In my career, I have tried a number of note-taking tools like OneNote, Evernote, CintaNotes, SimpleNote, Cherrytree, Google Keep, etc. But I have never been satisfied with them mainly because - 
 
 1. I don't have any control over how or where my notes are stored. - What if the company closes or the developer stops building the product?
-2. Most of these services are paid or work on only certain Operating Systems. And even if they are truly free and cross-platform, they lack critical features like fast full notebook search or hierarchical organization. 
```

Comment:
Was an extra space at the end intentional? :worried:

Reply:
No. I wasn't using `editorconfig`, hence this happened.